cola_inference.py

```diff
@@ -212,5 +254,40 @@ def take_one(ls):
     return ls[0]
 
 
+def load_cola_data(input_path, task, input_format, max_seq_len):
+    if input_format == "text":
+        with open(input_path, "r") as f_in:
+            sentences = f_in.readlines()
+        tokens = [
+            process_sentence(
+                tokenizer_name=task.tokenizer_name,
+                sent=sentence,
+                max_seq_len=max_seq_len,
+            )
+            for sentence in sentences
+        ]
+        labels = None
+    elif input_format == "train" or input_format == "dev":
+        data = load_tsv(task.tokenizer_name, input_path, max_seq_len,
+                        s1_idx=3, s2_idx=None, label_idx=1)
+        tokens, labels = data[0], data[2]
+    elif input_format == "test":
+        data = load_tsv(task.tokenizer_name, input_path, max_seq_len,
+                        s1_idx=1, s2_idx=None, has_labels=False, return_indices=True, skip_rows=1)
+        tokens, labels = data[0], None
+    else:
+        raise KeyError(input_format)
+    return tokens, labels
+
+
+def get_cola_metrics(logits, preds, labels, task):
```

Comment:
Ditto. Why does this need to be in this file?

Reply:
Similar to the above, the logic for computing metrics specific to CoLA is deeply embedded in the model code:

https://github.com/jsalt18-sentence-repl/jiant/blob/master/src/models.py#L672-L675

Unlike the above, I don't think there is currently a clean way to expose the same functionality. I recommend keeping this as is given the simplicity of the logic here, as well as the potential for adding additional metrics specifically for CoLA evaluation in the future.

Reply:
SGTM.